src/server/extension-context-builder.ts

```diff
@@ -0,0 +1,11 @@
+import { GraphQLExtension } from 'graphql-extensions';
+
+export class ExtensionContextBuilder extends GraphQLExtension {
+    willSendResponse(o: any) {
```

Comment:
the name of the method is a little weird.. willSendResponse sounds like a boolean method

Reply:
yep that was a little bit wierd.
But thats a method we need to overwrite from GraphQLExtension.
Thats our only way to format the response **and** access the context - according to a closed issue on apollo server repo